### PR TITLE
Add apport section to docs. Relates to #1488.

### PR DESCRIPTION
**Pull request by nickygerritsen**

### CODE REVIEW

**Review by vmcj — APPROVED** [review]

**Comment by vmcj** [comment]
Maybe actually also add this to the installation section? We have something on what the install with apt, and I think it makes sense if this is mentioned there also.

**Comment by nickygerritsen** [comment]
> Maybe actually also add this to the installation section? We have something on what the install with apt, and I think it makes sense if this is mentioned there also.

Added something. Not sure if the wording is OK.

**Review by thijskh — APPROVED** [review]

File: doc/manual/install-judgehost.rst [hunk 1]
```diff
@@ -40,6 +40,14 @@ For Red Hat::
 
 .. _installing-judgehost:
 
+Removing apport
+---------------
+
+Some systems (like Ubuntu) ship with ``apport``, which conflicts with judging.
+To uninstall uit, run::
```
Comment (eldering) [inline doc/manual/install-judgehost.rst:47]: Typo:
```suggestion
To uninstall it, run::
```

File: doc/manual/judging.rst [hunk 2]
```diff
@@ -215,3 +215,14 @@ create dedicated a user `domjudge` under which to install and run everything.
...
 
+'found processes still running ... apport'
+``````````````````````````````````````````
+
+If you see error messages of the form::
+
+  error: found processes still running as 'domjudge-run', check manually:
+  2342 apport
+
+Then you still have ``apport`` installed and/or running. This occurs on
```
Comment (eldering) [inline doc/manual/judging.rst:226]:
```suggestion
Then you still have ``apport`` installed and running. This error message occurs when
```

File: doc/manual/judging.rst [hunk 3]
```diff
@@ -215,3 +215,14 @@ create dedicated a user `domjudge` under which to install and run everything.
...
+submitting segmentation fault solutions. Disable or uninstall the apport
```
Comment (eldering) [inline doc/manual/judging.rst:227]:
```suggestion
judging submissions that trigger a segmentation fault. Disable or uninstall the apport
```

**Comment by nickygerritsen** [comment]
Will squash before merge btw

**Review by eldering — APPROVED** [review]

**Pull request merged**